moveit_ros/moveit_servo/src/collision_check.cpp

```diff
@@ -49,14 +49,16 @@ constexpr size_t ROS_LOG_THROTTLE_PERIOD = 30 * 1000;  // Milliseconds to thrott
 namespace moveit_servo
 {
 // Constructor for the class that handles collision checking
-CollisionCheck::CollisionCheck(const rclcpp::Node::SharedPtr& node, const servo::Params& servo_params,
-                               const planning_scene_monitor::PlanningSceneMonitorPtr& planning_scene_monitor)
+CollisionCheck::CollisionCheck(const rclcpp::Node::SharedPtr& node,
+                               const planning_scene_monitor::PlanningSceneMonitorPtr& planning_scene_monitor,
+                               std::shared_ptr<const servo::ParamListener> servo_param_listener)
   : node_(node)
-  , servo_params_(servo_params)
+  , servo_param_listener_(std::move(servo_param_listener))
+  , servo_params_(servo_param_listener_->get_params())
   , planning_scene_monitor_(planning_scene_monitor)
   , self_velocity_scale_coefficient_(-log(0.001) / servo_params_.self_collision_proximity_threshold)
   , scene_velocity_scale_coefficient_(-log(0.001) / servo_params_.scene_collision_proximity_threshold)
-  , period_(1. / servo_params_.collision_check_rate)
+  , period_(1.0 / servo_params_.collision_check_rate)
```

Comment:
This change seems extra, does it do something I don't understand?

Reply:
No, I just disliked that there was no number after the decimal point

Reply:
I don't care either way, just seemed odd to me. (I regularly just follow a number with a . to say it is a floating point but I could see how that syntax would look odd).